build/containermatrix-scenarios.yml

```diff
@@ -0,0 +1,85 @@
+# Blazor scenarios
+
+parameters:
+- name: server # the url of the agent service, e.g. http://asp-perf-lin:5001
+  type: string
+  default: ''
+- name: connection
+  type: string
+  default: ''
+- name: profile
+  type: string
+  default: ''
+
+# Sizes
+- name: sizes
+  type: object
+  default:
+
+    Mini:
+      displayName: Mini
+      cpu: 0.25
+      memory: 40mb
+    Small:
+      displayName: Small
+      cpu: 1
+      memory: 100mb
+    Medium:
+      displayName: Medium
+      cpu: 2
+      memory: 250mb
+    Large:
+      displayName: Large
+      cpu: 4
+      memory: 500mb
+    QuarterMachine:
+      displayName: QuarterMachine
+      cpu: 7
+      memory: 8gb
+    HalfMachine:
+      displayName: HalfMachine
+      cpu: 14
+      memory: 16gb
+    WholeMachine:
+      displayName: WholeMachine
+      cpu: 28
+      memory: 32gb
```

Comment:
@sebastienros I deduced from looking at TE's site that the citrine machine has 32GB of memory. True?

Reply:
correct